proposals/spi-proposal.md

```diff
@@ -89,11 +88,9 @@ them separately by not specifying a CS pin and instead using Firmata
 9:  maxSpeed              (bits 29 - 32)
 10: wordSize              (0 = DEFAULT = 8-bits, 1 = 1-bit, 2 = 2 bits, etc)
 11: csPin                 [optional] (0-127) The chip select pin number
-12: csPinOptions          (required if csPin specified)
-                          bit 0: CS_ACTIVE_STATE (0 = Active LOW (default)
+12: csPinOptions          bit 0: CS_ACTIVE_STATE (0 = Active LOW (default)
                                                   1 = Active HIGH)
-                          bit 1: CS_TOGGLE (0 = toggle at end of transfer (default)
```

Comment:
I don't know of any specific SPI devices that needed the CS_TOGGLE option, it was just a suggestion based on an old SPI thread. I'm removing it for now and reserving the additional bits for potential future options.

Reply:
This sounds like a good idea to me.